Sources/SotoCore/Doc/Region.swift

```diff
@@ -80,8 +80,8 @@ public struct Region: RawRepresentable, Equatable {
     public static func other(_ name: String) -> Region { .init(rawValue: name) }
 }
 
-extension Region {
-    public var partition: AWSPartition {
+public extension Region {
+    var partition: AWSPartition {
```

Comment:
Any reason you changed this?

Reply:
swiftformat did

Reply:
Have you upgraded to swift format v0.47? Soto is still using v0.46.3.

Reply:
I did, it was part of a a `brew upgrade`. would you recommend to stick to 0.46.3 ?

Reply:
There are differences between the output of 0.46.3 and 0.47. You might have issue submitting code, if the outputs disagree. I'll upgrade in the near future, but for the moment I'm sticking to 0.46.3